app/controllers/api/base_controller.rb

```diff
@@ -107,6 +107,15 @@ def initialize
     end
 
     before_action :parse_api_request, :log_api_request, :validate_api_request, :validate_api_action
+    before_action :validate_response_format, :only => [:show]
```

Comment:
I agree - could I do this in a follow up PR so as to keep this a straightforward refactoring?